tests/test_humann3.py

```diff
@@ -33,11 +33,12 @@ def test_humann3_m8():
     assert hash.unordered(output_pathcoverage_path) == 1315086232
 
 
-@pytest.mark.integration_full
+@pytest.mark.integration
 def test_humann3_fastq():
     """
     Tests humann3 with a fastq.gz file
-    Note: This test takes about 1 hour and should be run on pre-deploy tests only.
+    Note: This test uses taxonomic profile (from running the same input without one) to speed up execution. This skips
+    some steps that would otherwise happen but cause the test to take around an hour.
```

Comment:
How long does the new test take?

Reply:
Took < 5 minutes if I remember correctly from last week.